exercises/chapter7/src/Data/AddressBook/Validation.purs

```diff
@@ -64,16 +66,34 @@ lengthIs _     _   value = pure value
 -- ANCHOR_END: lengthIs
 
 -- ANCHOR: phoneNumberRegex
```

Comment:
Turns out this anchor is unused. Otherwise, I'd recommend moving the below comments next to the snippet in the text (instead of having the comments picked-up in the book's code snippets). No change necessary.

Reply:
So... no change necessary means we're good to go right? :)